news/2018-12-18-osu-world-cup-2018-recap.md

```diff
@@ -0,0 +1,83 @@
+
+![](/wiki/shared/news/banners/OWC_2018_banner.png)
+
+In a distinct departure from other years, the osu! World Cup (or OWC) featured its own Grand Finals specific mappool this year, where the playing field was bumped up a notch in difficulty again, forcing players to revise their strategies and picks from the Finals matches.
+
+### [United Kingdom vs Germany](https://osu.ppy.sh/community/matches/48120274)
+
+The first two teams up to play were Germany and the United Kingdom. 
+
+Led by veteran and star players [Dustice](https://osu.ppy.sh/users/754565) and [Okinamo](https://osu.ppy.sh/users/3765989), the German players were already set on making history - returning once again to the Top 3 for the first time since 2014. Their core skillsets were as solid as ever, and their fabled consistency showed no signs of slowing down or giving way. However, standing in their way was Team United Kingdom, gathered under perhaps the strongest roster of players they had ever fielded in a World Cup, who were absolutely hell-bent on becoming the second European team to be crowned since Poland's dominating performance in the OWC last year. 
+
+All-star players Bubbleman and Spare held strong as the backbone their team as the English players charged at the Germany's picks. Despite being constantly under pressure, the Germans held their ground, desperately not wanting to repeat their historical defeat from last year against the same opponent. 
+
+Despite their valiant efforts, Team Germany could just not muster enough strength to impede Team UK's unstoppable advance, though there were many instances where it seemed as if Germany was about to flip the match on its head and send the English running for their money. 
+
+In the end, Team Germany conceded defeat at 4-7. Nonetheless, this still served as a historical milestone for the German players, and will surely ignite their ambition to overcome more obstacles and climb higher in future OWCs.
+
+![](/wiki/shared/news/2018-12-18-osu!-world-cup-2018-recap/uk-de.jpg)
+
+### [United States vs United Kingdom](https://osu.ppy.sh/community/matches/48146823)
+
+The stage then was set for the true Grand Finals to take place on December 16th, 2018, where two-time champion Team USA squared up against Team UK in a contest for their third ever osu!Champion title. Over recent years, the United States' osu! tournament scene has seen an astronomical expansion since its advent in 2015, with new and skilled players showing up like mushrooms after rains in a forest.
```

Comment:
```suggestion
The stage then was set for the true Grand Finals to take place on December 16th, 2018, where two-time champion Team USA squared up against Team UK in a contest for their third ever osu!Champion title. Over recent years, the United States' osu! tournament scene has seen an astronomical expansion since its advent in 2015, with new and skilled players sprouting up like mushrooms after rains in a forest.
```